src/HL-phpclient/HLWebhooks.php

```diff
@@ -1,5 +1,5 @@
 <?php namespace Phpclient;
-use Exception;
+use \Exception;
```

Comment:
Note that for namespaced names (fully qualified namespace names containing namespace separator, such as Foo\Bar as opposed to global names that do not, such as FooBar), the leading backslash is unnecessary and not recommended, as import names must be fully qualified, and are not processed relative to the current namespace.